GliaWidgetsTests/ChatMessage/ChatMessageTests.swift

```diff
@@ -23,4 +23,15 @@ final class ChatMessageTests: XCTestCase {
         )
         XCTAssertEqual(msg.cardType, .customCard)
     }
+
```

Comment:
Maybe it makes sense to also add test for `. choiceCard` as well, since it is a part of the changes?